packages/dotcom-ui-client-embed/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './components/ClientEmbed'
+export * from './client'
```

Comment:
I've had to add the client entrypoints to the main entrypoints which I assume is not ideal since we have a separate browser entrypoint for a reason. This was necessary to allow the client code to be shared in other packages with the TypeScript types working...